t/venom/scenarios/pfappserver/playbooks/run_tests.yml

```diff
@@ -0,0 +1,35 @@
+---
+- hosts: pfservers
+  name: Run HTML pfappserver configurator tests
+  become: True
+
+  vars:
+    test_suites:
+    - pfappserver_configurator
+
+  tasks:
+    - name: Install Cypress
+      community.general.make:
+        chdir: /usr/local/pf/t/html/pfappserver
+        target: install
+        file: /usr/local/pf/t/html/pfappserver/Makefile
+      become: true
+      tags: cypress
```

Comment:
I suggest to move this task directly in `configure.yml` playbook.

Reply:
For some reason the cypress binary gets removed and needs to be reinstalled before each test. It's been moved from configure to here intentionally to work-around this issue.